src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -83,19 +83,52 @@ function WorkspaceInvitePage(props) {
     const excludedUsers = useMemo(() => PolicyUtils.getIneligibleInvitees(props.policyMembers, props.personalDetails), [props.policyMembers, props.personalDetails]);
 
     useEffect(() => {
-        const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, props.betas, searchTerm, excludedUsers);
-
-        // Update selectedOptions with the latest personalDetails and policyMembers information
-        const detailsMap = {};
-        _.forEach(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
-        const newSelectedOptions = [];
-        _.forEach(selectedOptions, (option) => {
-            newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+        let emails = searchTerm.replace(/\s,\s/g, ',').split(',');
+        emails = _.map(emails, word => word.trim());
+
+        const newUsersToInviteDict = {};
+        const newPersonalDetailsDict = {};
+        const newSelectedOptionsDict = {};
+
+        _.forEach(emails, (email) => {
+            if (email === '') {
```

Comment:
Can we move this to `_.map` line adding a filter?

Reply:
You happy with this? `emails = _.filter(_.map(emails, word => word.trim()), email => email !== '');`. If so I will make the changes now.

Reply:
Yeah this works.  Once you do this, I'll pull the changes, test and complete the checklist.

I was also checking if we should make a util for this, but I don't think it's worth the time.

Reply:
> Yeah this works. Once you do this, I'll pull the changes, test and complete the checklist.
> 
> I was also checking if we should make a util for this, but I don't think it's worth the time.

Should be fine for now, it is only one line. Something to think about. I've pushed the commit.

Reply:
I meant util for the whole search, filter, etc. along with the usersToInvite. But as I said ignore for now.